docs/content/reference/versioning-support.md

```diff
@@ -1,42 +1,91 @@
-# Versioning support
-
+# Versioning Support
 There are different components that might require independent versioning and support level:
 
-- API `hypershift.openshift.io`.
-- HyperShift Operator (HO).
-- Control Plane Operator (CPO).
-- HyperShift CLI.
+- Management Cluster
+- API `hypershift.openshift.io`
+- HyperShift Operator (HO)
+- Control Plane Operator (CPO)
+- HyperShift CLI
+- Hosted Control Plane (HCP) CLI
+
+## Support Level
+### Management Cluster
+The upstream HyperShift project does not place strict requirements on the OpenShift version of your management cluster. The HO determines what versions of OCP can be installed through the HostedCluster (HC); see the [HO section](#ho) for more details. However, different versions of the HO are thoroughly tested only on a limited set of OpenShift versions, and this should inform your deployment decisions.
```

Comment:
> The upstream HyperShift project does not place strict requirements on the OpenShift version of your management cluster

This is not entirely true. There are API versions that we use in our kube clients that require a particular version of Kube/OCP. There's also providers like Kubevirt which require 4.14 OCP.